salt/apache/files/docs/error_page.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
We'd probably do something more like this (removing the meta-refresh, inlining the minimum styles, and changing the message): https://github.com/open-contracting/extension-explorer/blob/main/extension_explorer/templates/redirect.html